sections/errorhandling/returningpromises.md

```diff
@@ -0,0 +1,94 @@
+# Returning promises
+
+<br/>
+
+### One Paragraph Explainer
+
+The mechanisms behind sync function stacktrace and async function stacktrace in v8 implementation are quite different:
+sync stacktrace is based on **stack** provided by operation system Node.js is running on (just like in most programming
```

Comment:
We could use the plural when talking about the stacktraces, something like: `sync stacktraces are based on the stack provided by`

I think we should say `the operating system` instead of `operation` here and in the other sentences

Reply:
I have tried to pluralize "stacktraces", but with it I needed to pluralize "functions", "stacks", "promises", "chains" etc., and when everything is plural it's not clear that all relations between entities are always 1-to-1, so the text becomes less univocal and I decide to pluralize only the first part of the first sentence